.github/workflows/ci.yml

```diff
@@ -0,0 +1,143 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  tutorials_set_1:
+    name: Tutorials1 ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.5'
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+      - uses: actions/cache@v1
+        env:
+          cache-name: cache-artifacts
+        with:
+          path: ~/.julia/artifacts
+          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
+          restore-keys: |
+            ${{ runner.os }}-test-${{ env.cache-name }}-
+            ${{ runner.os }}-test-
+            ${{ runner.os }}-
+      - uses: julia-actions/julia-buildpkg@v1
+      - run: cd test/; julia --project=.. --color=yes --check-bounds=yes runtests.jl poisson.jl validation.jl elasticity.jl 
```

Comment:
`runtests.jl` runs all tests. Not needed here, right?

idem in the other jobs.

Reply:
> `runtests.jl` runs all tests. Not needed here, right?
> 
> idem in the other jobs.

Yes, you are right. I thought it was working when I went to bed yesterday, but it wasn't. 

Solved in ca0ecccdd02e67c30cb5bc63f975b2cf2b75735b. Note that the idea now is that `runtests.jl` (optionally) gets command-line arguments with the particular set of tutorial filenames to be executed.  

With parallel jobs, although there is a little bit of job unbalance (I tried to balance the load manually by looking at the computational times), now it takes around 7 minutes to execute all tutorials. Not that bad ...

Reply:
> 7 minutes to execute all tutorials.

Great job!